docs/usage.md

```diff
@@ -39,6 +39,27 @@ nf-core/funcscan utilises various tools that use databases and reference files t
 
 Here we will describe where you can obtain databases and reference files for tools included in the pipeline.
 
+### Bakta
+
+nf-core/funcscan offers multiple tools for annotating input sequences.
+
+Bakta is a new tool touted as a bacteria-only successor to the well-established Prokka.
+
+Bakta requires a database, supplied to the pipeline with `--annotation_bakta_db`.
+This must be downloaded from the Bakta Zenodo archive, the link of which can be found on the [Bakta github repository](https://github.com/oschwengers/bakta).
+
+Once downloaded this must be untarred
+
+```bash
+tar xvzf db.tar.gz
+```
+
+And then passed to the pipeline with
+
+```bash
+--annotation_bakta_db /<path>/<to>/db/
+```
+
```

Comment:
Yes... I sort of feel this will take a bit of time and isn't a blocker for a release for funcscan, so I suggest if I run out of tasks I will look into it. I'll make an issue.

Reply:
Ok yes, but we defniintely cannot  run this in the tests:

```
All database releases (latest 4.0, 31 Gb zipped, 59 Gb unzipped) are hosted at Zenodo: DOI
```

59GB but the the github runners are 14GB HDD space :sweat_smile: